client/src/components/BrowseChefs/Chefs.js

```diff
@@ -0,0 +1,73 @@
+import React from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import { Grid, Paper, Box, Avatar, Typography, Chip } from "@material-ui/core";
+
+const Chefs = ({ chefProfileUrl, name, location, cuisineTags, bio }) => {
+  const useStyles = makeStyles((theme) => ({
+    avatar: {
+      width: "100px",
+      height: "100px",
+    },
+
+    cuisineChip: {
+      borderRadius: "0px",
+    },
+
+    chefBio: {
+      display: "box",
+      lineClamp: 2,
+      boxOrient: "vertical",
+      overflow: "hidden",
+    },
+  }));
+
+  const classes = useStyles();
+
+  const mapCuisineTags = (cuisineTags) => {
+    if (cuisineTags.length > 0) {
+      return cuisineTags.map(
+        (cuisineTag) => `${cuisineTag.toUpperCase()} CUISINE`
```

Comment:
This displays very strangely when there are multiple tags. Maybe it would make sense to use one chip per cuisine? I also wonder if appending CUISINE is necessary or if it clutters up the UI (not sure if that's to match the mocks or not, maybe you can discuss with the team)

Reply:
Now returning one cuisine per chip. Have to append CUISINE to match the mockup.